### ISSUE TITLE
Form control label class

### ISSUE THREAD

**Issue by leobewater** [realripley00/bootstrap-form#5]
The form control label class should be 'col-form-label' instead of 'form-control-label'

```
    protected function getLabelOptions(array $options = [])
    {
        $class = 'col-form-label';
        if ($this->isHorizontal()) {
            $class .= ' ' . $this->getLeftColumnClass();
        }

        return array_merge(['class' => trim($class)], $options);
    }
```

**Comment by realripley00** [comment]
Fixed on v 1.1.25

**Comment by SystemF** [comment]
For non horizontal forms (BootForm::open()) the 'col-form-label' class should not be appended.
